mne/evoked.py

```diff
@@ -116,6 +121,11 @@ def save(self, fname):
         ----------
         fname : string
             Name of the file where to save the data.
+            
+        Notes
+        -----
+        To write multiple conditions into a single file, use 
+        :func: `mne.write_evokeds()`.
```

Comment:
Sorry for rushing a bit and not RTFM. I fixed it, but now AppVeyor fails for some reason that seems to be unrelated to the diff.